web/e2e-tests/drafts.test.ts

```diff
@@ -177,7 +177,7 @@ async function test_restore_private_message_draft_via_draft_overlay(page: Page):
     await common.pm_recipient.expect(page, `${cordelia_internal_email},${hamlet_internal_email}`);
     assert.strictEqual(
         await common.get_text_from_selector(page, "title"),
-        "Cordelia, Lear's daughter, King Hamlet - Zulip Dev - Zulip",
+        "Cordelia, Lear's daughter and King Hamlet - Zulip Dev - Zulip",
```

Comment:
This hunk should presumably be squashed into whatever commit caused the result of this test to change.

Reply:
It was caused due to ```web/src/people.ts```. Squashed with ```people.ts``` commit.